pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -2310,18 +2248,6 @@ fn overload_settle_on_block() {
                 alice_init_balance - 1500
             );
             assert_eq!(Asset::balance_of(&ticker, bob_did), bob_init_balance + 1500);
-            assert_eq!(
-                Settlement::scheduled_instructions(block_number + 3).len(),
-                0
-            );
-            assert_eq!(
-                Settlement::scheduled_instructions(block_number + 4).len(),
-                0
-            );
-            assert_eq!(
-                Settlement::scheduled_instructions(block_number + 5).len(),
-                0
-            );
```

Comment:
Can we add relevant asserts to make sure that the schedule has been created?

Reply:
@maxsam4 Implemented, please check. The actual content of the `Agenda` vectors cannot be pattern-matched because the value `struct` in `pallet_scheduler` does not publish its fields either directly or via getters. So we can only measure the number of scheduled calls.